m4/cf3_platforms.m4

```diff
@@ -39,3 +39,4 @@ AM_CONDITIONAL([HPUX], [test -n "`(echo ${target_os} | egrep 'hpux|hp-ux')`"])
 AM_CONDITIONAL([FREEBSD], [test -n "`(echo ${target_os} | grep freebsd)`"])
 AM_CONDITIONAL([NETBSD], [test -n "`(echo ${target_os} | grep netbsd)`"])
 AM_CONDITIONAL([XNU], [test -n "`(echo ${target_os} | grep darwin)`"])
+AM_CONDITIONAL([SHUNIT], [test -n "`which shunit2`"])
```

Comment:
Here is a snippet from the failing test on Solaris 9:

```
11:11:30 /opt/csw/bin/gmake   test_package_module_freebsd_ports
11:11:30 gmake[2]: Entering directory `/home/jenkins/workspace/testing-enterprise-pr_masterfiles/label/PACKAGES_ULTRASPARC_SOLARIS_9/masterfiles/tests/unit'
11:11:30 gmake[2]: Nothing to be done for `test_package_module_freebsd_ports'.
11:11:30 gmake[2]: Leaving directory `/home/jenkins/workspace/testing-enterprise-pr_masterfiles/label/PACKAGES_ULTRASPARC_SOLARIS_9/masterfiles/tests/unit'
11:11:30 /opt/csw/bin/gmake  check-TESTS
11:11:30 gmake[2]: Entering directory `/home/jenkins/workspace/testing-enterprise-pr_masterfiles/label/PACKAGES_ULTRASPARC_SOLARIS_9/masterfiles/tests/unit'
11:11:30 ./test_package_module_freebsd_ports: PATH=tests/stubs:/usr/bin:/opt/csw/bin:/opt/sfw/bin:/home/jenkins/workspace/testing-enterprise-pr_masterfiles/label/PACKAGES_ULTRASPARC_SOLARIS_9/./buildscripts/build-scripts:/var/cfengine/httpd/php/bin:/usr/local/bin:/usr/sbin:/usr/sfw/bin:/usr/bin:/usr/local/sbin:/usr/contrib/bin:/home/jenkins/workspace/testing-enterprise-pr_masterfiles/label/PACKAGES_ULTRASPARC_SOLARIS_9/./buildscripts/build-scripts:/var/cfengine/httpd/php/bin:/usr/local/bin:/usr/sbin:/usr/sfw/bin:/usr/bin:/usr/local/sbin:/usr/contrib/bin:/home/jenkins/workspace/testing-enterprise-pr_masterfiles/label/PACKAGES_ULTRASPARC_SOLARIS_9/buildscripts/build-scripts:/var/cfengine/httpd/php/bin:/usr/local/bin:/usr/sbin:/usr/sfw/bin:/usr/bin:/usr/local/sbin:/usr/contrib/bin:/home/jenkins/workspace/testing-enterprise-pr_masterfiles/label/PACKAGES_ULTRASPARC_SOLARIS_9/buildscripts/build-scripts:/var/cfengine/httpd/php/bin:/usr/local/bin:/usr/sbin:/usr/sfw/bin:/usr/bin:/usr/local/sbin:/usr/contrib/bin: is not an identifier
11:11:30 FAIL: test_package_module_freebsd_ports
11:11:30 ==================
11:11:30 1 of 1 test failed
11:11:30 ==================
11:11:30 gmake[2]: *** [check-TESTS] Error 1
11:11:30 gmake[2]: Leaving directory `/home/jenkins/workspace/testing-enterprise-pr_masterfiles/label/PACKAGES_ULTRASPARC_SOLARIS_9/masterfiles/tests/unit'
11:11:30 gmake[1]: *** [check-am] Error 2
11:11:30 gmake[1]: Leaving directory `/home/jenkins/workspace/testing-enterprise-pr_masterfiles/label/PACKAGES_ULTRASPARC_SOLARIS_9/masterfiles/tests/unit'
11:11:30 gmake: *** [check-recursive] Error 1
11:11:30 gmake: Leaving directory `/home/jenkins/workspace/testing-enterprise-pr_masterfiles/label/PACKAGES_ULTRASPARC_SOLARIS_9/masterfiles'
```

And from HP-UX 11.23:

```
11:49:30 /usr/local/bin/gmake   test_package_module_freebsd_ports
11:49:30 gmake[2]: Entering directory '/home/jenkins/workspace/testing-enterprise-pr_masterfiles/label/PACKAGES_ia64_hpux_11.23/masterfiles/tests/unit'
11:49:30 gmake[2]: Nothing to be done for 'test_package_module_freebsd_ports'.
11:49:30 gmake[2]: Leaving directory '/home/jenkins/workspace/testing-enterprise-pr_masterfiles/label/PACKAGES_ia64_hpux_11.23/masterfiles/tests/unit'
11:49:30 /usr/local/bin/gmake  check-TESTS
11:49:30 gmake[2]: Entering directory '/home/jenkins/workspace/testing-enterprise-pr_masterfiles/label/PACKAGES_ia64_hpux_11.23/masterfiles/tests/unit'
11:49:30 ./test_package_module_freebsd_ports[165]: no:  not found.
11:49:30 FAIL: test_package_module_freebsd_ports
11:49:30 ==================
11:49:30 1 of 1 test failed
11:49:30 ==================
11:49:30 Makefile:201: recipe for target 'check-TESTS' failed
11:49:30 gmake[2]: *** [check-TESTS] Error 1
11:49:30 gmake[2]: Leaving directory '/home/jenkins/workspace/testing-enterprise-pr_masterfiles/label/PACKAGES_ia64_hpux_11.23/masterfiles/tests/unit'
11:49:30 Makefile:324: recipe for target 'check-am' failed
11:49:30 gmake[1]: *** [check-am] Error 2
11:49:30 gmake[1]: Leaving directory '/home/jenkins/workspace/testing-enterprise-pr_masterfiles/label/PACKAGES_ia64_hpux_11.23/masterfiles/tests/unit'
11:49:30 Makefile:311: recipe for target 'check-recursive' failed
11:49:30 gmake: *** [check-recursive] Error 1
11:49:30 gmake: Leaving directory '/home/jenkins/workspace/testing-enterprise-pr_masterfiles/label/PACKAGES_ia64_hpux_11.23/masterfiles'
```

I would really recommend making this specific to FreeBSD, unless you have the time and equipment to investigate and solve this test failure yourself. Since the package module is for a platform we don't officially support, I'm afraid I cannot invest time in this myself.